lib/codegen/meta-python/cdsl/instructions.py

```diff
@@ -108,6 +109,7 @@ class Instruction(object):
             'True for all indirect branch or jump instructions.',
             'is_call': 'Is this a call instruction?',
             'is_return': 'Is this a return instruction?',
+            'is_ghost': 'Is this a ghost instruction?',
```

Comment:
It'd be nice to precise what it means: my understanding is that it's an instruction with an illegal encoding that should not be visited by register allocation. (fallthrough_return has an illegal encoding but must be visited -- are there any other ones like fallthrough_return?)